Resources/Prototypes/Entities/Clothing/Shoes/specific.yml

```diff
@@ -9,8 +9,27 @@
   - type: Clothing
     sprite: Clothing/Shoes/Specific/chef.rsi
 
+# stuff common to all clown & jester shoes
 - type: entity
-  parent: ClothingShoesBaseButcherable
+  abstract: true
+  parent: [ClothingShoesBaseButcherable, ClothingSlotBase]
+  id: ClothingShoesClownBase
+  components:
+  - type: ItemSlots
+    slots:
+      item:
+        name: clothing-boots-sidearm
+        whitelist:
+          components:
+          - Utensil
+          tags:
+          - ToySidearm
+        blacklist:
+          components:
+          - MeleeWeapon
```

Comment:
im pretty sure plastic utensils are still low damage MeleeWeapons, whats the point of the utensil tag with this

Reply:
simply make plastic utensils not do damage

Reply:
Nope, UtensilBase doesn't have the MeleeWeapon component.

I can rearrange utensils.yml if we should make it more clear which is what?

Reply:
if no utensils like plastic knife fork spoon etc have MeleeWeapon then its fine

Reply:
New blacklist should cover both.